Test_Automation_Framework/EPAM_TestAutomation_Tests/EpamSiteNavigateTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Epam_TestAutomation_Core.Browser;
+using Epam_TestAutomation_Core.Helper;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Epam_TestAutomation_Tests
+{
```

Comment:
First of all you have to rewrite the tests you wrote earlier for Selenium lessons.
The second solution doesn't work. The problem is with config file and TestSettings.cs classes.